tools/coqdep/lib/common.ml

```diff
@@ -126,30 +111,30 @@ module VCache = Set.Make(VData)
     (those loaded by [Require]) from other dependencies, e.g. dependencies
     on ".v" files (for [Load]) or ".cmx", ".cmo", etc... (for [Declare]). *)
 
-let legacy_mapping = Core_plugins_findlib_compat.legacy_to_findlib
+let warn_legacy_loading =
+  let name = "legacy-loading-removed-coqdep" in
+  CWarnings.create ~name (fun name ->
+      Pp.(str "Legacy loading plugin method has been removed from Coq, \
+               and the `:` syntax is deprecated, and its first \
+               argument ignored; please remove \"" ++
+          str name ++ str ":\" from your Declare ML"))
```

Comment:
Will it be clear what the terminology "legacy loading plugin method" means? Would there be a way to describe it without using a word whose meaning is intrinsically unstable over time.

Reply:
It seems clear from context that it means the one using `:` no?

Reply:
We can reconstruct from the context that there is indeed an history of loading methods and that one of them used `:'`.

I'm playing the candid here, but in which sense (from the user point of view), is the new loading method essentially a new method rather than just a modification of the method?

Depending on the answer, a possible suggestion is:
```suggestion
      Pp.(strbrk "Declare ML Module now ignores the \"" ++ \
          str name ++ str ":\" part of its argument; \
          this part is deprecated, please remove it."))
```